### PR TITLE
Updated required versions in Stimfit linux install

### PR DESCRIPTION
**Pull request by JoseGuzman**
I indicated the equirements for wxWidgets = 2.8, matplotlib => 1.5, Numpy =>1.7 etc. Otherwise the installation will not work, or will return the famous error when loading the Python shell.

### CODE REVIEW

**Comment by neurodroid** [comment]
wxWidgets 3.x should be fine? That's what I use on most systems nowadays.

**Comment by JoseGuzman** [comment]
Sure, which one would you recommend?

Jose

**Comment by neurodroid** [comment]
I'd say that `>= 2.8` should do the job? The problem with the original version (`= 2.8`) is that it suggests that only 2.8 will work.

**Comment by JoseGuzman** [comment]
> I'd say that `>= 2.8` should do the job? The problem with the original version (`= 2.8`) is that it suggests that only 2.8 will work.

Thanks, I was missing the ">=" symbols and wasn't sure which 3.0 version of wxWidgets was working. I tested now the installation for versions **3.0.4** and **2.8.12** and that they both work under Linux.

File: doc/sphinx/linux_install_guide/requirement.rst [hunk 1]
```diff
@@ -91,8 +94,7 @@ to generate the configure script. Remember that we need Autoconf, Automake and L
 
     $ ./configure --enable-python
 
-The **--enable-python** option is absolutely necessary to install `Stimfit <http://www.stimfit.org>`_ since some of the functionality depends on Python. The configure script has some additional options. For example, we may want to use `IPython <http://www.scipy.org>`_  instead of the default embedded python shell with the option **---enable-ipython**  (note that the `IPython <http://www.scipy.org>`_ shell is only available under GNU/Linux and it is still very experimental). 
-
+The **--enable-python** option is absolutely necessary to install `Stimfit <http://www.stimfit.org>`_ since some of the functionality depends on Python. The configure script has some additional options.
```
Comment (neurodroid) [inline doc/sphinx/linux_install_guide/requirement.rst]: Isn't `enable-python` activated by default? No need to explicitly add it in that case.

File: doc/sphinx/linux_install_guide/requirement.rst [hunk 2]
```diff
@@ -153,7 +155,7 @@ After that you can enter the option --with-biosig in the configure script of `St
 Building documentation
 ======================
 
-The manual of `Stimfit <http://www.stimfit.org>`_ including the documentation is accessible on-line in http://www.stimfit.org/doc/sphix/. To have your local copy, you will need to install sphinx:
+The manual of `Stimfit <http://www.stimfit.org>`_ including the documentation is accessible on-line in http://www.stimfit.org/doc/sphix/. To have your local copy, you will need to install sphinx version 1.7 or older:
```
Comment (neurodroid) [inline doc/sphinx/linux_install_guide/requirement.rst]: Are you sure that it has to be "older"?

File: doc/sphinx/linux_install_guide/requirement.rst [hunk 3]
```diff
@@ -36,19 +36,22 @@ For the impatient, here are all `Stimfit <http://www.stimfit.org>`_ build depend
 
 This will get you, amongst others:
 
+* [wxWidgets]_: C++ library that contains the user interface (version >= 2.8). Tested with 2.8.12 and 3.0.4)
```
Comment (neurodroid) [inline doc/sphinx/linux_install_guide/requirement.rst]:
```suggestion
* [wxWidgets]_: C++ graphical user interface toolkit (version >= 2.8). Tested with 2.8.12 and 3.0.4)
```

File: doc/sphinx/linux_install_guide/requirement.rst (hunk 1 above)
Comment (JoseGuzman) [reply doc/sphinx/linux_install_guide/requirement.rst]: I wasn't aware of that. Please, remove it.

File: doc/sphinx/linux_install_guide/requirement.rst (hunk 2 above)
Comment (JoseGuzman) [reply doc/sphinx/linux_install_guide/requirement.rst]: I tested 1.4.1 and returned an error. Better add  version =>1.7

Comment (neurodroid) [reply doc/sphinx/linux_install_guide/requirement.rst]: > Better add version =>1.7

That's version 1.7 _or newer_.
Greater than or equal is denoted by `>=`, _not_ `=>`. Please make sure it's the right way around in your pull request.

**Pull request merged**